sys/ztimer64/overhead.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2020 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2020 Freie Universität Berlin
+ *               2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     sys_ztimer_overhead
+ * @{
+ *
+ * @file
+ * @brief       ztimer overhead measurement functions
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include "ztimer64.h"
+
+typedef struct {
+    ztimer64_clock_t *clock;
+    volatile uint64_t *val;
```

Comment:
Does this need to be volatile? Can't atomics be used? Or `irq_disab/enable`?

Reply:
ah, I just copied that from the 32bit version, but for 64bit, this probably breaks anyways. will fix

Reply:
done, using atomics now.

Reply:
ah, it's still a volatile, as the `atomic_load/store_u64()` take ptr to volatile as args.